src/components/DynamicFormContainer/DynamicForm/AttachmentDropzone/FilesInfo/FilesInfo.test.tsx

```diff
@@ -3,7 +3,7 @@ import React from "react";
 import { FilesInfo } from "./FilesInfo";
 
 describe("filesInfo", () => {
-  it("should render a uploaded file correctly", () => {
+  it("should render a uploaded pdf file correctly", () => {
```

Comment:
can we keep the previous test? then the additional ones we test only for icon:

```
it("should render a pdf icon correctly", () => {
    render(
      <FilesInfo
        filesInfo={[
          {
            filename: "asdfdfs.pdf",
            data: "asdfasdf",
            type: "application/pdf",
          },
        ]}
        removeFile={() => {}}
      />
    );
    expect(screen.queryByTestId("attachment-icon-application/pdf")).not.toBeNull();
  });

```

^ maybe will be clearer on whats the intention of the test + expected result, i got abit confused by the repeated expects